notebooks/official/model_monitoring/model_monitoring.ipynb

```diff
@@ -202,7 +202,6 @@
       "outputs": [],
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #24.    ! rm -f /opt/conda/lib/python3.7/site-packages/tensorflow/core/kernels/libtfkernel_sobol_op.so
</pre><p>What does this line do? Removed a package? Why?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/841/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>relic from a prior era, fixed.</p>